src/plugins/paginate/paginate-en.hbs

```diff
@@ -0,0 +1,839 @@
+---
+{
+	"title": "Paginate",
+	"language": "en",
+	"category": "Plugins",
+	"description": "Adds pagination at the bottom of a list of items",
+	"tag": "paginate",
+	"parentdir": "paginate",
+	"altLangPrefix": "paginate",
+	"dateModified": "2023-05-15"
+}
+---
+
+<div class="alert alert-warning">
+	<h2>Unstable feature</h2>
+	<p>To be used at <strong>your own risk</strong>. This feature described below can be removed in any subsequent
+		minor/major release</p>
+	<p><small><a href="https://wet-boew.github.io/wet-boew-documentation/decision/7.html">Learn more about the design
+				decision around provisional features.</a></small></p>
+	<p><small><a href="../../docs/ref/provisional-en.html">Check other provisional features available.</a></small>
+	</p>
+</div>
+
+<span class="wb-prettify all-pre"></span>
+
+<h2>On this page:</h2>
+<ul>
+	<li><a href="#paginatedList">Example 1 - List</a></li>
+	<li><a href="#paginatedTable">Example 2 - Table</a></li>
+	<li><a href="#paginatedArticles">Example 3 - Articles</a></li>
+	<li><a href="#filteredEventsList">Example 4 - Events list</a></li>
+</ul>
+
+<section id="paginatedList">
+	<h2>Example 1 - List</h2>
+	<ul class="wb-paginate provisional">
+		<li class="wb-pgfltr-out">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Illum, at!</li>
```

Comment:
Is the initial state is hidden? I suggest to do the reverse that way if there is a JS error in the plugin, the content will still be displayed, as per the progressive enhancement design approach.